src/Mvc/Mvc.Analyzers/src/Startup/StartupAnalysisContext.cs

```diff
@@ -7,6 +7,7 @@ namespace Microsoft.AspNetCore.Analyzers
 {
     internal class StartupAnalysisContext
     {
+#pragma warning disable RS1012 // Start action has no registered actions.
```

Comment:
The 3.0 Roslyn package comes with some new rules for analyzers/codefixes. 

I have another change planned that avoids this warning so suppressing it for now.